File: 1.cpp
#include <iostream>
#include <vector>

class Solution
{
public:
    std::vector<int> twoSum(std::vector<int>& nums, int target)
    {
        bool bFound = false;
        int tobeZero = target;
        std::vector<int> vOutput;
        int i,j;
        for(i=0; i<nums.size(); ++i)
        {
            tobeZero -= nums[i];
            for(j=i+1; j<nums.size(); ++j)
            {
                if(tobeZero == nums[j])
                {
                    bFound = true;
                    break;
                }
            }
            if(bFound == true)
            {
                break;
            }
            else
            {
                tobeZero = target;
            }
        }
        vOutput.push_back(i);
        vOutput.push_back(j);
        return vOutput;
    }
};

File: 1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res = []
        for i in range(len(nums)):
            # Select ith number
            select = nums[i]
            # Find if target - nums[i] is there.
            for j in range(i+1, len(nums)):
                
                # if Found, return
                if nums[i] + nums[j] == target:
                    res.append(i)
                    res.append(j)
                    return res
                # else, loop through again